client/public/recipes/recipe_details.html

```diff
@@ -434,6 +434,10 @@ <h3>Dietary Labels</h3>
           <!-- Dynamically building dietary labels here -->
         </ul>
       </section>
+      
+      <form id="updateRecipeForm">
+        <section id="updateRecipeContainer"></section>
```

Comment:
What does this form look like if this is a non user recipe? Also, is this just a button similar to the tags below?

Reply:
In a non user recipe it doesn't even show up. The section can be seen on the inspection of the html page, but there is no button. 
non-user recipe:
![image](https://github.com/user-attachments/assets/dbc14224-d790-4e3f-adfa-c4700a940d75)
user recipe:
![image](https://github.com/user-attachments/assets/beebc45d-1284-48e3-b426-b2ca331bc757)

Reply:
I'm thinking future we want to have these buttons more flush as in less space between the two buttons